content/zh-cn/docs/contribute/_index.md

```diff
@@ -157,9 +157,9 @@ Figure 1 outlines a roadmap for new contributors. You can follow some or all of
 图 1. 新手入门指示。
 
 图 1 概述了新贡献者的路线图。
-你可以遵从`注册`和`评审`所述的某些或全部步骤。
+你可以遵从“注册”和“评审”所述的某些或全部步骤。
 至此，你完成了发起 PR 的准备工作，
-可以通过`发起 PR`列出的事项实现你的贡献目标。
+可以通过“发起 PR” 列出的事项实现你的贡献目标。
```

Comment:
```suggestion
可以通过 “发起 PR” 列出的事项实现你的贡献目标。
```

Reply:
I am prefer to add a space before `“”` too. But in this project, it seems not encouraged to add that space.

CC @tengqm could you explain more on this?